conftest.py

```diff
@@ -75,6 +75,25 @@ def fix_missing_hostname():
             logger.error("Failed to add hostname")
 
 
+@pytest.fixture(scope="session", autouse=True)
+@pytest.mark.uncollectif(lambda: version.current_version() < "5.6")
+def patch_appliance():
+    """ Patch appliance over SSH"""
+    logger.info('Patching appliance')
+    # (local_path, remote_path, md5/None) trio
+    patch_args = (
+        (str(data_path.join('miq_appliance.js.diff')),
+         '/var/www/miq/vmdb/app/assets/javascripts/miq_appliance.js',
+         None),
+        (str(data_path.join('autofocus.js.diff')),
+         '/var/www/miq/vmdb/app/assets/javascripts/directives/autofocus.js',
+         'f5ce9fa129d1662e6fe6f7c213458227'),
+    )
+    ssh_client = store.current_appliance.ssh_client
+    for local_path, remote_path, md5 in patch_args:
+        ssh_client.patch_file(local_path, remote_path, md5)
+
```

Comment:
We are gonna have to collect if there was any patching done here and potentially precompile assets. (wip)